### PR TITLE
Use generic CLI commands on upgrade tests

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: templates/github/.github/workflows/scripts/before_install.sh.j2 [hunk 1]
```diff
@@ -111,8 +111,7 @@ fi
 
 cd pulp-cli
 pip install -e .
-pulp config create --base-url http://pulp --location tests/settings.toml
-sed -i "s/true/false/g" tests/settings.toml
+pulp config create --base-url http://pulp --location tests/settings.toml --no-verify-ssl
```
Comment (mdellweg) [inline templates/github/.github/workflows/scripts/before_install.sh.j2:114]: :100:

File: templates/github/.github/workflows/scripts/script.sh.j2 [hunk 2]
```diff
@@ -146,11 +146,8 @@ if [[ "$TEST" == "upgrade" ]]; then
 
   # CLI commands to display plugin versions and content data
   pulp status
-  {%- if plugin_name != 'pulpcore' %}
-  pulp {{ plugin_app_label }} content list
-  {%- else %}
-  pulp file content list
-  {%- endif %}
+  pulp show --href /pulp/api/v3/content/
```
Comment (mdellweg) [inline templates/github/.github/workflows/scripts/script.sh.j2:149]: This is not a supported command.
show is meant to be used with detail views. And we will probably add some validation here that will break this usage.

Comment (fao89) [reply templates/github/.github/workflows/scripts/script.sh.j2:149]: it worked with pulp_ansible, what is the best generic command that I can put here?
See previous discussion: https://github.com/pulp/pulp_ansible/pull/576#discussion_r636197472

Comment (mdellweg) [reply templates/github/.github/workflows/scripts/script.sh.j2:149]: Well this is nothing more than the plain `http $HOST/pulp/api/v3/content/`. And also it's not supported and may break in the future. To list content, you'd need to specify the type like `pulp file content list` (where "content" is the content type unit).
Also a generic content list command is just a few lines of CLI code away.

Quite honestly i'm mostly wondering what this command is supposed to test. Calling out to the generic content endpoint and not even looking at the result seems rather pointless to me.

Comment (fao89) [reply templates/github/.github/workflows/scripts/script.sh.j2:149]: it is for debugging purposes, I wanted to answer the following questions:
`pulp status` - are pulpcore and plugins really upgraded?
`pulp content list` - did we keep the data after the migration?

Comment (mdellweg) [reply templates/github/.github/workflows/scripts/script.sh.j2:149]: https://github.com/pulp/pulp-cli/pull/254

Here you go.

**Review by mdellweg — APPROVED** [review]

**Pull request merged**